meilisearch-core/src/facets.rs

```diff
@@ -158,30 +161,31 @@ pub fn add_to_facet_map(
 ) -> Result<(), FacetError> {
     let value = match value {
         Value::String(s) => s,
-        Value::Object(_) | Value::Array(_) => return Err(FacetError::InvalidDocumentAttribute(document_id)),
+        Value::Object(_)
+        | Value::Array(_)
+        | Value::Number(_) => return Err(FacetError::InvalidDocumentAttribute(document_id)),
         v => v.to_string(),
     };
```

Comment:
I think we must accept only strings and array or strings? Here you are converting Null and Booleans to strings, but that's invalid.

Reply:
I think filtering on a bool is a very common and legitimate thing to do, we should allow that. And we probably should not error out on null, is is very common to have null values where something else is expected. maybe ignoring them is better suited if we don't want them around

Reply:
Yeah,, good point, we should ignore nulls but I maintain that we must only support strings and arrays of string, this PR will just support those two types only. We will support numbers including booleans in a next PR.